scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -586,6 +588,36 @@ class ScioContext private[scio] (val options: PipelineOptions,
     }
 
   /**
+    * Get an SCollection for a CloudSql query
+    * @group input
+    */
+  def cloudSqlSelect[T: ClassTag](readOptions: JdbcReadOptions[T]): SCollection[T] = {
```

Comment:
> CloudSQL nothing but mysql

Imprecise: https://cloud.google.com/sql/docs/postgres/

It would be nice to support postgres also.